.github/workflows/e2e-tests-windows.yml

```diff
@@ -32,7 +32,11 @@ jobs:
       - run: echo $(./.github/workflows/scripts/is_changed_file_windows.sh ${{ github.event.pull_request.base.sha }} ${{ github.sha }} )
 
   collector-build:
-    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-2022, windows-2025]
```

Comment:
Actually this one is fine to use windows-latest since it is building the binary, we should be fine with `windows-latest` - it also breaks because it tries to upload two artifacts with the same name.